homeassistant/components/wiz/translations/hu.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
These will disappear one rebase. Its just out of sync with `dev` because of the dep chain